Please add the following patch to the Fedora kernel (F-7, FC-6 & devel).
This was originally in the FC-6 kernel but seems to have vanished. This
patch isn't going to get sent upstream for various reasons but we need
it in Fedora so that the GFS kernel stuff will build against the GFS2
lock modules.

Steve.

-----------------------------------------------------------------------------
--- linux-2.6.17.noarch/fs/gfs2/locking.c~	2006-08-10 13:33:09.000000000 -0400
+++ linux-2.6.17.noarch/fs/gfs2/locking.c	2006-08-10 13:33:23.000000000 -0400
@@ -188,4 +188,6 @@ void __init gfs2_init_lmh(void)
 
 EXPORT_SYMBOL_GPL(gfs2_register_lockproto);
 EXPORT_SYMBOL_GPL(gfs2_unregister_lockproto);
-
+EXPORT_SYMBOL_GPL(gfs2_withdraw_lockproto);
+EXPORT_SYMBOL_GPL(gfs2_mount_lockproto);
+EXPORT_SYMBOL_GPL(gfs2_unmount_lockproto);


